### DISCUSSION TITLE
How to extend tsconfig genereted by nuxi?

### DISCUSSION THREAD

**Discussion by Rusinas** [nuxt/nuxt#18616] [Questions] [answered]
I've created Nuxt 3 project and it has tsconfig at the project root, which extends another one in the .nuxt folder. Generated tsconfig has a lot of paths specified (a lot of useful stuff like #app, #head and other things). I don't like auto-imports and use those paths a lot, but my project is in monorepo and in order to catch all my libraries, I have to provide my custom paths (something like ~libs/ui for example), otherwise autocompletion will not work.

Huge bummer is that you can't merge two tsconfigs, you can only overwrite fields in existing one.

**Comment by mklueh** [comment, marked as answer]
I had the same problem, see  https://github.com/nuxt/nuxt/issues/12386

**Reply by Rusinas** [reply]
Whoa! Thank you so much, that was fast 🥹 And solved my problem too
